### PR TITLE
Replaces choice field segment status with boolean

### LINKED ISSUE #149: Replace `Segment.status` with an `enabled` boolean
If we don’t need more statuses than just enabled/disabled, we should replace this with a boolean.

### PR DESCRIPTION
**Pull request by jberghoef**
Fixes #149

### CODE REVIEW

File: src/wagtail_personalisation/migrations/0013_auto_20170609_1321.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
```
Comment (blurrah) [inline src/wagtail_personalisation/migrations/0013_auto_20170609_1321.py]: Can you rename this migration to describe what it is changing?

File: src/wagtail_personalisation/models.py [hunk 2]
```diff
@@ -101,9 +93,9 @@ def get_rules(self):
         return segment_rules
 
     def toggle(self, save=True):
-        self.status = (
-            self.STATUS_ENABLED if self.status == self.STATUS_DISABLED
-            else self.STATUS_DISABLED)
+        self.enabled = (
```
Comment (blurrah) [inline src/wagtail_personalisation/models.py]: Might be able to shorten this to `self.enabled = not self.enabled` but not sure if that is good practice.

Comment (jberghoef) [inline src/wagtail_personalisation/models.py]: Explicit before implicit right?

File: src/wagtail_personalisation/migrations/0021_segment_status_as_boolean.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wagtail_personalisation', '0020_rules_delete_relatedqueryname'),
+    ]
+
+    operations = [
```
Comment (loicteixeira) [inline src/wagtail_personalisation/migrations/0021_segment_status_as_boolean.py]: Status data will be lost without a multi-step migration which:

1. create the new field
1. populate the new field from the old one
1. delete the old field
